tasks/testTasks.ts

```diff
@@ -12,7 +13,8 @@ gulp.task('test:e2e', async () => {
     const testRepoPath = path.join(testAssetsPath, "vscode-docs-build-e2e-test");
 
     // Initialize Submodule
-    cp.execSync('git submodule update --init', { cwd: rootPath });
+    const git = simpleGit(rootPath);
+    await git.submoduleInit();

```

Comment:
`await git.submoduleUpdate(['--init']);`
init won't update the submodule, the test also fails since this repository hasn't been restored successfully.

Reply:
Switched to your form, although it looks like i'm getting my original odd failure with either approach again